include/drivers/keyboard.h

```diff
@@ -0,0 +1,12 @@
+#ifndef DRIVER_KEYBOARD_H_
+# define DRIVER_KEYBOARD_H_
+
+# include <chaosdef.h>
+
+# define INT_KEYBOARD			0x21
+# define KEYBOARD_IO_PORT		0x60
```

Comment:
This should be defined in `include/arch/x86/asm.h` instead.

Reply:
done.